### PR TITLE
#0: LLM Tech Report: Intro

### PR DESCRIPTION
**Pull request by yieldthought**
Adds the intro section for the tech report doc

### CODE REVIEW

File: tech_reports/LLMs/llms.md [hunk 1]
```diff
@@ -37,6 +37,17 @@ Authors:
         - [4.10.4.2 Large Matmuls](#41042-large-matmuls)
 
 ## 1. Overview
+What’s the best way to bring up high-performance multi-chip models on Tenstorrent hardware with the Metal stack? We continue to try to answer this question; here are our current best practices, tips and workarounds.
+
+What you need:
+
+* Access to TT hardware. This guide is specifically for bringing models up on wormhole, so whilst most of this advice applies equally to grayskull it is very WH (wormhole)-centric.
```
Comment (mtairum) [inline tech_reports/LLMs/llms.md:44]: * Access to TT hardware. This guide is specifically for bringing models up on wormhole (WH), so whilst most of this advice applies equally to grayskull it is very WH-centric.

File: tech_reports/LLMs/llms.md [hunk 2]
```diff
@@ -37,6 +37,17 @@ Authors:
         - [4.10.4.2 Large Matmuls](#41042-large-matmuls)
 
 ## 1. Overview
+What’s the best way to bring up high-performance multi-chip models on Tenstorrent hardware with the Metal stack? We continue to try to answer this question; here are our current best practices, tips and workarounds.
```
Comment (mtairum) [inline tech_reports/LLMs/llms.md:40]: Chaging the tone of the intro a bit:

This document aims to provide guidance on how to bring up high-performance multi-chip models on Tenstorrent hardware using the TT-Metal stack.
It is targeted at users with previous experience on TT-Metal and shares our current best practices, tips, caveats and workarounds on model bringup.

File: tech_reports/LLMs/llms.md [hunk 3]
```diff
@@ -37,6 +37,17 @@ Authors:
...
+
+What you need:
+
+* Access to TT hardware. This guide is specifically for bringing models up on wormhole, so whilst most of this advice applies equally to grayskull it is very WH (wormhole)-centric.
+* Good grasp of PyTorch and transformers. We will not be covering any basics here. For example, this document assumes you understand what a kv-cache is and get the difference between prefill (reading tokens and generating the kv-cache entries) and decode (auto-regressively generating new tokens one at a time). Beginner tutorials will follow, for now this is to help experts get up to speed deploying LLMs on Metal.
```
Comment (mtairum) [inline tech_reports/LLMs/llms.md:45]: `We will not be covering any basics here.` -> `This document will only skim some basics.`

File: tech_reports/LLMs/llms.md [hunk 4]
```diff
@@ -37,6 +37,17 @@ Authors:
...
+* Familiarity with Metal and ttnn. How to install, build, run examples and so on.
```
Comment (mtairum) [inline tech_reports/LLMs/llms.md:46]: Link to the installing guide in the main TT-Metal repo.

File: tech_reports/LLMs/llms.md [hunk 5]
```diff
@@ -37,6 +37,17 @@ Authors:
...
+* Will to win. This is the cutting edge, which means everything is still being developed and sometimes both the API and our advice will be in flux. It’s fun and everything is open source so if in doubt trust the code, but it’s not called the cuddling edge for a reason and not everything will always be easy! We will get there, though, and with your help all the faster.
```
Comment (mtairum) [inline tech_reports/LLMs/llms.md:47]: I don't dislike this one for now, but we need to come back to it depending on the overall tone of the full tech report at the end. 👍

File: tech_reports/LLMs/llms.md [hunk 6]
```diff
@@ -37,6 +37,17 @@ Authors:
...
+
+The [ViT guide](https://github.com/tenstorrent/tt-metal/blob/main/tech_reports/ViT-TTNN/vit.md) provides an excellent introduction to using Metal with transformers and if anything in this document seems unclear or intimidating you should look at that first. It has pictures!
```
Comment (mtairum) [inline tech_reports/LLMs/llms.md:49]: Good as well. Might be best to change it to another part of the intro when the full tech report is done.

**Review by mtairum — CHANGES_REQUESTED** [review]
1st paragraph I think it needs to setup a different tone. I'm thinking something like:

This document aims to provide users with past experience on TT-Metal

File: tech_reports/LLMs/llms.md (hunk 2 above)
Comment (uaydonat) [reply tech_reports/LLMs/llms.md:40]: Yes, I agree with comment.

File: tech_reports/LLMs/llms.md (hunk 3 above)
Comment (uaydonat) [reply tech_reports/LLMs/llms.md:45]: Yes. I agree with comment. We are briefly describing kv-cache, decode/prefill in other sections, but we do get into details.

File: tech_reports/LLMs/llms.md (hunk 5 above)
Comment (uaydonat) [reply tech_reports/LLMs/llms.md:47]: I think the language makes the software stack look less mature than it is. It is ok to mention some work is still in progress and some code may change, but the stack is mature and works (even Nvidia has bugs).

File: tech_reports/LLMs/llms.md (hunk 3 above)
Comment (uaydonat) [reply tech_reports/LLMs/llms.md:45]: I found this is a great intro. Maybe we should link it.
https://levelup.gitconnected.com/building-llama-3-from-scratch-with-python-e0cf4dbbc306

File: tech_reports/LLMs/llms.md (hunk 5 above)
Comment (yieldthought) [reply tech_reports/LLMs/llms.md:47]: Removed

**Review by mtairum — APPROVED** [review]

**Pull request merged**